tests/conftest.py

```diff
@@ -320,10 +319,13 @@ def aiohttp_client_cls():
     return CoalescingClient
 
 
+TestClientGenerator = Callable[..., Coroutine[Any, Any, TestClient]]
+
+
```

Comment:
Considering how often this is used in the tests, I think it makes sense to add the `Alias`.
But maybe this is not the best place for it?